ddtrace/tracer.py

```diff
@@ -661,7 +705,6 @@ def write(self, spans):
         if not self.enabled:
             return
 
-        spans = self._processor.process(spans)
         if spans is not None:
             self.writer.write(spans=spans)
```

Comment:
`SpanAggregator` also calls `_writer.write()` on span finish. When would that be called compared to `tracer.write()`?

Reply:
`tracer.write()` is actually no longer used by us in the library. It is left due to it being a public API and someone could be using it.

That said, you have raised a valid issue that tracer.write() will no longer have trace filters applied to it. This is technically a breaking change which we should have a release note for.

We should also deprecate `tracer.write()`